pkg/apis/serving/v1beta1/configmap.go

```diff
@@ -65,6 +65,8 @@ type PredictorConfig struct {
 	DefaultTimeout int64 `json:"defaultTimeout,string,omitempty"`
 	// Flag to determine if multi-model serving is supported
 	MultiModelServer bool `json:"multiModelServer,boolean,omitempty"`
+	// frameworks the model agent is able to run
+	SupportedFrameworks []string `json:"supportedFrameworks"`
```

Comment:
I am not sure what this `SupportedFrameworks` means, isn't there is already `MultiModelServer` flag on the given framework?